src/customerproductorder/Classes/Order.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package customerproductorder.Classes;
+import java.util.*;
+/**
+ *
+ * @author AS
+ */
+public class Order {
+    private Customer customer;
+    private ArrayList<Product> products;
```

Comment:
Use interface for collection var. Implementation will be only when you create a new List...